internal/models/objectmeta/object_status.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
I don't think this is a property of the object or its metadata, rather it is a resource status.

@Shubbhang351 Please consider moving it to a new package called `status`, we can add batch details and phase as well while adding fanout/BR status.